ansible/roles/test/tasks/copp.yml

```diff
@@ -1,13 +1,18 @@
+- include_tasks: add_container_to_inventory.yml
+  vars:
+    container_name: "{{ item }}"
+  with_items:
+    - "lldp"
+    - "syncd"
+
```

Comment:
Can we move this part into add_container_to_inventory, add with_items for all containers?

Reply:
I mean if we move every `include_task: add_container_to_inventory` into inside that file with all the known container names, we can keep other files clean, only include_tasks once.

---
In reply to: [356377776](https://github.com/Azure/sonic-mgmt/pull/1269#discussion_r356377776) [](ancestors = 356377776)

Reply:
We do not use it in all test cases, so I do not see the reason to implicitly add all containers to inventory.